### ISSUE TITLE
confidence interval in the regression.ipynb

### ISSUE THREAD

**Issue by AmosJoseph** [TyXe-BDL/TyXe#30]
Hi，in the regression.ipynb (https://github.com/TyXe-BDL/TyXe/blob/master/notebooks/regression.ipynb),

In [11] :

plt.scatter(x, y, color="black")
plt.plot(x_test, y_test, color="black")
plt.plot(x_test, m.detach(), color="blue")
for c in range(1, 4):
    plt.fill_between(x_test.squeeze(), (m - c * sd).squeeze(), (m + c * sd).squeeze(), alpha=c * 0.1, color="blue")
plt.ylim(-2, 2)

confidence interval can be got: (m - 3 * sd, m + 3 * sd).
But why? As the prediction may be not subject to Normal distribution.

Looking forward to your reply!
 
Best!
